src/frontend/components/VideoPlayer/index.spec.tsx

```diff
@@ -198,4 +198,49 @@ describe('VideoPlayer', () => {
     const videoElement = container.querySelector('video')!;
     expect(videoElement.tabIndex).toEqual(-1);
   });
+
+  it('displays the waiting message when a live is paused or stopping', async () => {
+    const pausedStates = [liveState.STOPPING, liveState.PAUSED];
+    const video = videoMockFactory({
+      live_state: pausedStates[Math.floor(Math.random() * pausedStates.length)],
+      urls: {
+        manifests: {
+          hls: 'https://example.com/hls.m3u8',
+        },
+        mp4: {},
+        thumbnails: {},
+      },
+      xmpp: {
+        bosh_url: 'https://xmpp-server.com/http-bind',
+        websocket_url: null,
+        conference_url:
+          '870c467b-d66e-4949-8ee5-fcf460c72e88@conference.xmpp-server.com',
+        prebind_url: 'https://xmpp-server.com/http-pre-bind',
+        jid: 'xmpp-server.com',
+      },
+    });
+
+    const { container } = render(
+      wrapInIntlProvider(
+        <VideoPlayer
+          video={video}
+          playerType={'videojs'}
+          timedTextTracks={[]}
+        />,
+      ),
+    );
+    await waitFor(() =>
+      // The player is created and initialized with DashJS for adaptive bitrate
+      expect(mockCreatePlayer).toHaveBeenCalledWith(
+        'videojs',
+        expect.any(Element),
+        expect.anything(),
+        video,
+      ),
+    );
+
+    const videoElement = container.querySelector('video')!;
+    expect(videoElement.tabIndex).toEqual(-1);
+    screen.getByText('Webinar is paused');
+  });
 });
```

Comment:
maybe we could add another test proving that in the other tests screen.getByText('Webinar is paused'); is not displayed.